extract_data/metadata.py

```diff
@@ -728,6 +728,61 @@ def add_missing_references_to_paper_collection():
                         paper["pubmed_id"]["value"],
                     )
 
+def manually_add_paper(dict):
+    metadata = {}
+    all_authors = []
+
+    title_dict = {
+        "value": dict["title"],
+        "attr_id": 139
+    }
+    metadata["title"] = title_dict
+
+    counter = 1
+    for author in dict["authors"].items():
```

Comment:
In the example you sent, `authors` was defined as follows:
```python
       'authors': {
                       1: {'last_name': 'McDougal', 'first_name': 'Robert', 'initials': 'RA', 'orcid': None}, 
                       2: {'last_name': 'Zhu', 'first_name': 'Goldie', 'initials': 'G', 'orcid': None}
        },
```

Would it suffice to make this a list of authors? If so, then we could turn this into:
```
for author in dict["authors"]:
```

and in the insde all the `dict["authors"][counter]` become just `author`, and we can eliminate the `counter` variable.